modules/300-prometheus/templates/memcached/servicemonitor.yaml

```diff
@@ -12,6 +12,10 @@ spec:
     - port: http-metrics
       scheme: http
       honorLabels: true
+      metricRelabelings:
+        - action: drop
+          regex: (memcached_slab_.*)
+          sourceLabels:
```

Comment:
Missing name for source label. It needs to be like this:
```yaml
       - action: drop
          regex: 'memcached_slab_.*'
          sourceLabels: [__name__]
```

Reply:
My bad. I fixed a mistake.